web/crux-ui/src/validations/container.ts

```diff
@@ -207,7 +207,8 @@ const createOverlapTest = (
   )
 
 const portConfigRule = yup.mixed().when('portRanges', portRanges => {
```

Comment:
It seems to me that the actual fix is to just wrap the portRanges in an array like `([portRanges]) => ...`.
Because yup passes all references fields to the function, in this case there's only one reference but it still seems to pass it as an array.

Reply:
Well, this is the cleanest.